ruby/game/hangman.rb

```diff
@@ -0,0 +1,83 @@
+# make a class
+class Game
+# add attributes for target_word, guessed_letters, guessed_word, number_of_guesses
+	attr_reader :number_of_guesses
+	attr_accessor :target_word, :guessed_letters, :guessed_word
+
+# add methods for getting the target_word, guessing letters
+	# initalize setting the guessed_letters and guessed_word to empty arrays
+	def initialize
+		puts "You have started a game of hangman!"
+		@guessed_letters = []
+		@guessed_word = []
+	end
+
+	# get the target word from the user, set the number of guesses and build the guessed_word
+	def target(word)
+		@target_word = word
+		word.length.times {@guessed_word <<  '_'}
+		@number_of_guesses = word.length * 2
```

Comment:
number_guesses is doing something way different than setting a target.  it should be its own method